packages/components/roving-focus-group/src/roving-focus-group-impl.vue

```diff
@@ -148,15 +147,11 @@ export default defineComponent({
       }
     )
 
-    onMounted(() => {
-      const rovingFocusGroupEl = unref(rovingFocusGroupRef)!
-      on(rovingFocusGroupEl, ENTRY_FOCUS_EVT, handleEntryFocus)
-    })
-
-    onBeforeUnmount(() => {
-      const rovingFocusGroupEl = unref(rovingFocusGroupRef)!
-      off(rovingFocusGroupEl, ENTRY_FOCUS_EVT, handleEntryFocus)
-    })
+    useEventListener(
+      rovingFocusGroupRef.value,
```

Comment:
```suggestion
      rovingFocusGroupRef,
```

Support passing a ref